hoodie-client/src/main/java/com/uber/hoodie/config/HoodieWriteConfig.java

```diff
@@ -369,6 +377,42 @@ public double getLogFileToParquetCompressionRatio() {
     return Double.valueOf(props.getProperty(HoodieStorageConfig.LOGFILE_TO_PARQUET_COMPRESSION_RATIO));
   }
 
+  // orc
+  public String getOrcStringSchema() {
+    return props.getProperty(HoodieStorageConfig.ORC_STRING_SCHEMA);
+  }
+
+  public CompressionKind getOrcCompressionKind() {
+    String compressionName = props.getProperty(HoodieStorageConfig.ORC_COMPRESS);
+    switch (compressionName.toUpperCase()) {
```

Comment:
instead of this switch case, can we do `CompressionKind.valueOf()`, which should achieve the same?

Reply:
+1

Reply:
@vinothchandar @n3nash  I fixed this, thanks